autoPlay.user.js

```diff
@@ -1238,6 +1244,18 @@ function isRainingRound(level)
 	}
 }
 
+function isWormholeRound(level)
+{
+	var mod = level % WORMHOLE_INFO.LEVEL;
+
+	if (mod === 0) {
+		return true;
```

Comment:
Samething, mine is just verbose to combine with the other functions that does the same kinda of stuff.
Semantics...